packages/json/tests/json/convert-to-map/base-case.test.js

```diff
@@ -23,5 +23,5 @@ test('json | convert-to-map > base case', function () {
     ['ember-test-selectors', '^6.0.0'],
   ]);
 
-  assert.deepEqual(convertToMap(devDependencies), expectedValue);
+  assert.deepStrictEqual(convertToMap(devDependencies), expectedValue);
```

Comment:
While documenting `@codemod-utils/tests`, I learned that `assert.deepEqual` is an alias of `assert.deepStrictEqual` and the latter should be used to avoid unexpected results.

> **Strict assertion mode**
>
> An alias of [assert.deepStrictEqual()](https://nodejs.org/api/assert.html#assertdeepstrictequalactual-expected-message).
>
> **Legacy assertion mode**
> 
> [Stability: 3](https://nodejs.org/api/documentation.html#stability-index) - Legacy: Use [assert.deepStrictEqual()](https://nodejs.org/api/assert.html#assertdeepstrictequalactual-expected-message) instead.
>
> Tests for deep equality between the actual and expected parameters. Consider using [assert.deepStrictEqual()](https://nodejs.org/api/assert.html#assertdeepstrictequalactual-expected-message) instead. [assert.deepEqual()](https://nodejs.org/api/assert.html#assertdeepequalactual-expected-message) can have surprising results.

https://nodejs.org/api/assert.html#assertdeepequalactual-expected-message